ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py

```diff
@@ -248,6 +248,53 @@ def test_linear(
     )
 
 
+# alpha_dropout
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.common.alpha_dropout",
+    d_type_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        shared_dtype=True,
+        min_value=2,
+        max_value=5,
+        min_num_dims=1,
```

Comment:
The function doesn't only work for `1-Dim` Arrays we need to test other dimensions as well